### ISSUE TITLE
Add a test case for ID20230630074506

### ISSUE THREAD

**Issue by eggrobin** [unicode-org/unicodetools#506]
From properties#149, [ID20230630074506](https://www.unicode.org/review/pri469/feedback.html#ID20230630074506): we should ensure that we have a grapheme segmentation test case with an ExtCccZwj just after the second LinkingConsonant in a three-consonant cluster:

LinkingConsonant ExtCccZwj\* ConjunctLinker ExtCccZwj\* LinkingConsonant **ExtCccZwj** ConjunctLinker ExtCccZwj\* LinkingConsonant

**Comment by markusicu** [comment]
Wasn't it about a *trailing* ExtCccZwj?

**Comment by eggrobin** [comment]
No, Charlotte’s suggestion would have added a trailing ExtCccZwj, but that is covered by postcore anyway.

The real bug was « we get a sequence pattern where ExtCccZwj can occur only after a ConjunctLinker but not before it: […] This does not match rule GB9c which accounts for ExtCccZwj in both positions, which is necessary because Indic scripts make use of combining marks with CCC values both smaller and greater than 9 (Virama). »
